packages/astro/src/core/config/schema.ts

```diff
@@ -318,6 +318,7 @@ export const AstroConfigSchema = z.object({
 								.or(z.custom<ShikiTheme>())
 						)
 						.default(ASTRO_CONFIG_DEFAULTS.markdown.shikiConfig.themes!),
+					defaultColor: z.union([z.string(), z.literal(false)]).optional(),
```

Comment:
```suggestion
					defaultColor: z.union([z.literal('light'), z.literal('dark'), z.string(), z.literal(false)]).optional(),
```

Should we also specify the `light` and `dark` literals here?